src/awssdk_driver.rs

```diff
@@ -47,6 +48,10 @@ impl MapRequest for UserAgent {
             Ok(req)
         })
     }
+
+    fn name(&self) -> &'static str {
```

Comment:
Is it used anywhere?

Reply:
It's part of the trait implementation. It's used for instrumenting tracing.